GUI/PyQt/createJson.py

```diff
@@ -8,6 +8,22 @@
 wFile = json.dumps(workspace)
 with open('lastWorkspace.json', 'w') as json_data:
     json_data.write(wFile)
+
+labels = {'names':['label_1', 'label_2', 'label_3', 'label_4', 'label_5', 'label_6', 'label_7', 'label_8','label_9','label_10','label_11', 'label_12', 'label_13', 'label_14', 'label_15', 'label_16', 'label_17', 'label_18','label_19','label_20','label_21', 'label_22', 'label_23', 'label_24', 'label_25', 'label_26', 'label_27', 'label_28','label_29','label_30'],
+           'colors':['red', 'orange', 'yellow', 'green', 'cyan', 'blue', 'purple', 'pink','olive', 'brown', 'darkred ' , 'darkorange', 'lime', 'teal', 'darkcyan', 'indigo', 'plum', 'steelbule','tomato', 'honeydew','rosybrown', 'coral', 'peru', 'peachpuff', 'sienna', 'darkblue', 'slategrey', 'navy', 'orchid', 'crimson'],
+          'path':['I:/CNNArt-PyQt_GUI_zIWEI/GUI/PyQt/Markings']}
+wFile = json.dumps(labels)
+with open('editlabel2.json', 'w') as json_data:
+    json_data.write(wFile)
+
+# label
+labels = {'names':{'list':['label_1', 'label_2', 'label_3', 'label_4', 'label_5', 'label_6', 'label_7', 'label_8','label_9','label_10','label_11', 'label_12', 'label_13', 'label_14', 'label_15', 'label_16', 'label_17', 'label_18','label_19','label_20','label_21', 'label_22', 'label_23', 'label_24', 'label_25', 'label_26', 'label_27', 'label_28','label_29','label_30']},
+          'colors':{'list':['red', 'orange', 'yellow', 'green', 'cyan', 'blue', 'purple', 'pink','olive', 'brown', 'darkred ' , 'darkorange', 'lime', 'teal', 'darkcyan', 'indigo', 'plum', 'steelbule','tomato', 'honeydew','rosybrown', 'coral', 'peru', 'peachpuff', 'sienna', 'darkblue', 'slategrey', 'navy', 'orchid', 'crimson']},
+          'layer':{}}
+wFile = json.dumps(labels)
+with open('I:/CNNArt-PyQt_GUI_zIWEI/GUI/PyQt/Markings/01_ab.json', 'w') as json_data:
```

Comment:
This is very specific.... Is the file generated generically?

Reply:
The color of the label is setting in this json file , each label corresponding to a color, after that the user can add label directly without to set different color.